merp_picking_list_sorted/models/stock.py

```diff
@@ -12,7 +12,7 @@ def sort_printer_picking_list(self, move_line_ids):
         """ sort list of pack operations by configured field
         """
         strategy = self.env.user.company_id.outgoing_routing_strategy
-        strategy_order = self.env.user.company_id.outgoing_routing_order
+        strategy_order = int(self.env.user.company_id.outgoing_routing_order)
```

Comment:
@Werded,  it will not work with Product based sorting. Check with Denis about this

Reply:
fixed